api/tests/routes/pro/post_offer_test.py

```diff
@@ -288,6 +292,29 @@ def test_create_collective_offer_on_draft_offer_creation(self, app):
         assert collective_offer.contactPhone == data["extraData"]["contactPhone"]
         assert collective_offer.offerVenue == data["extraData"]["offerVenue"]
 
+    def test_withdrawable_event_offer_can_have_no_ticket_to_withdraw(self, app):
+        # Given
+        venue = offers_factories.VenueFactory()
+        offerer = venue.managingOfferer
+        offers_factories.UserOffererFactory(offerer=offerer, user__email="user@example.com")
+
+        # When
+        data = {
+            "venueId": humanize(venue.id),
+            "name": "La pièce de théâtre",
+            "subcategoryId": subcategories.CONCERT.id,
+            "withdrawalDetails": "Veuillez récuperer vos billets à l'accueil :)",
+            "withdrawalType": "no_ticket",
+        }
+        client = TestClient(app.test_client()).with_session_auth("user@example.com")
```

Comment:
`TestClient` c'est tellement 2020 ! :) Avec la fixture `client` (à mettre en argument de la méthode de test, à la place de `app`) : 

```suggestion
        client = client.with_session_auth("user@example.com")
```

(idem dans le test plus bas)

Reply:
Après toutes ces heures passées à coder ensemble sur un-éditeur-douteux-dont-je-tairai-le-nom-de-Code, tu utilises `TestClient` @bruno-ebstein-pass-culture ? 😭

Reply:
je l'ai fait dans tout les tests du fichier

Reply:
Pas très explicite comme raccourci. Moi j'ai `<virgule>dblr` pour ça. Ou `:dbatyLeRageux` quand j'ai envie d'appuyer sur plein de touches pour simuler l'utilisation d'un éditeur tentaculaire.